src/queries.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const node = require('./graphql').queryNode;
+
+const authoredQ = node('nodes')
+			.addChild(node('author')
+								.addChild(node('login'))
+								.addChild(node('url')))
+			.addChild(node('createdAt'));
+
+const participantsQ = authoredQ
+			.addChild(node('comments', {first: 100})
+								.addChild(node('totalCount'))
+								.addChild(node('pageInfo')
+													.addChild(node('endCursor')))
+								.addChild(authoredQ));
+
+const multiQs = [
+	node('totalCount'),
+	node('pageInfo').addChild(node('endCursor')),
+	participantsQ
+];
+
+const everything = (repoName, ownerName) =>
+			node('repository', {name: repoName, owner: ownerName})
+			.addChild(node('pullRequests', {first: 100}, multiQs))
+			.addChild(node('issues', {first: 100}, multiQs));
+
+const timeFilter = (before = new Date(), after = new Date(0)) =>
+			data => data.filter(x => after <= x.createdAt <= before);
+
+const users = arr =>
+	arr.map(x => x.author)
+			// Get rid of null authors (weird GitHub problem)
+			// Deleted accounts maybe?
+			.filter(x => !(null === x || undefined === x))
+			.map(x => [x.login, x.url]);
+
+const flatten = arr => arr.reduce((acc, next) => acc.concat(next), []);
```

Comment:
Doesn't lodash have this as a method? import it directly?

Reply:
My general philosophy is that if something takes less than a minute to implement, downloading, and learning a library is a negative time gain. It also makes the package size way bigger.